kpi/tests/api/test_api_environment.py

```diff
@@ -109,14 +111,19 @@ def test_template_context_processor(self):
 
     def test_mfa_help_text_default_translation(self):
         MOCK_TRANSLATION_STRING = 'hello from gettext'
+        messages_dict = to_python_object(
+            constance.config.MFA_LOCALIZED_HELP_TEXT
+        )
+        default_translation = messages_dict['default']
+
         with mock.patch(
             'hub.utils.i18n.t', return_value=MOCK_TRANSLATION_STRING
         ) as mock_t:
             response = self.client.get(self.url, format='json')
             assert response.json()['mfa_localized_help_text'] == markdown(
                 MOCK_TRANSLATION_STRING
             )
-            assert mock_t.call_args.args[0] == MFA_DEFAULT_HELP_TEXT
+            assert mock_t.call_args.args[0] == default_translation._proxy____args[0]
```

Comment:
looks scary! can we coerce `default_translation` to a `str` to achieve the same?

Reply:
I wanted to be sure the behaviour of this test some the same as before. If we coerce `default_translation` to as `str`, it does not give the exact result. 

```python 
In [1]: from django.utils.translation import activate, gettext_lazy as t

In [2]: activate('fr')

In [3]: s = t('Create an account')

In [4]: str(s)
Out[4]: 'Ouvrir un compte'

In [5]: s._proxy____args[0]
Out[6]: 'Create an account'
```

Reply:
But if you don't `activate('fr')`, then `str(default_translation)` would work? I changed it locally and the test passes.

However, the whole point might be moot. Do we even need to examine what was passed as the argument to `t()` anymore now that you've gotten away from my split-brain thing of duplicating the string in `static_lists.py`?

Reply:
> str(default_translation) would work?

Indeed 

> Do we even need to examine what was passed as the argument to t()

Well, we may not need to test anymore.